kafka-ui-api/src/test/java/com/provectus/kafka/ui/container/KafkaConnectContainer.java

```diff
@@ -12,8 +11,7 @@ public class KafkaConnectContainer extends GenericContainer<KafkaConnectContaine
   public KafkaConnectContainer(String version) {
     super("confluentinc/cp-kafka-connect:" + version);
     addExposedPort(CONNECT_PORT);
-    waitStrategy = Wait.forHttp("/")
-        .withStartupTimeout(Duration.ofMinutes(5));
+    Wait.forHealthcheck();
```

Comment:
It's more complicated as that, I tried this but this wasn't enough to make it work, probably because KC container's healthcheck fails for something :)